blog/2015/11/applications-for-december.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "Applications for December Instructor Training Are Now Closed"
+date: 2015-11-20
+time: "01:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+  Applications to take part in
+  <a href="{{page.root}}/blog/2015/10/call-for-instructor-training.html">December's two-day instructor training class</a>
+  are now closed.
+  We received more than three dozen applications from four continents,
+  and will let people know early next week whether they have been selected.
+  Groups that we can't include in this round will be given priority to take part in the new year.
+</p>
+<!-- end excerpt -->
+<p>
+  Meanwhile,
+  yesterday saw the final meeting of the six-week cohort of instructor trainees.
+  The next step is to match trainees with experienced instructors
+  who will guide them through the core Data Carpentry and Software Carpentry material
+  and oversee their final mini-lesson.
```

Comment:
The mentoring and mini-lesson will happen only in January for people that conclude both version of instructor training.